packages/camera/camera_platform_interface/lib/src/types/resolution_preset.dart

```diff
@@ -5,22 +5,26 @@
 /// Affect the quality of video recording and image capture:
 ///
 /// If a preset is not available on the camera being used a preset of lower quality will be selected automatically.
+/// If the [CaptureMode] is [CaptureMode.photo] the selected resolution will be a 4:3 resolution.
+/// If the [CaptureMode] is [CaptureMode.video] the selected resolution will generally be a 16:9 resolution.
```

Comment:
Is this guaranteed on all platforms?

Reply:
For my implementations on Android and iOS yes. But feasibly It could be different in other platforms based off the camera sensors. As a side note, just like with the native cameras on most modern phones, when your device is in photo mode you often have the option to swap between 4:3, 16:9 and 1:1. So I might remove this comment.